.github/workflows/ci.yml

```diff
@@ -12,6 +12,16 @@ jobs:
   build:
     name: Build
     runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - arch: amd64
+            output: scoop-search.exe
+          - arch: 386
+            output: scoop-search-x86.exe
```

Comment:
If we are to release the 32bit version I think it would be more appropriate to be more explicit and suffix it with `-x86_32` or `-i386`

Reply:
Bump